pandas/tests/test_interval.py

```diff
@@ -63,10 +63,79 @@ def test_hash(self):
         # should not raise
         hash(self.interval)
 
-    # def test_math(self):
-    #     expected = Interval(1, 2)
-    #     actual = self.interval + 1
-    #     self.assertEqual(expected, actual)
+    def test_math_add(self):
+        expected = Interval(1, 2)
+        actual = self.interval + 1
+        self.assertEqual(expected, actual)
+
+        expected = Interval(1, 2)
+        actual = 1 + self.interval
+        self.assertEqual(expected, actual)
+
+        actual = self.interval
+        actual += 1
+        self.assertEqual(expected, actual)
+
+        with self.assertRaises(TypeError):
+            self.interval + Interval(1, 2)
+
+    def test_math_sub(self):
+        expected = Interval(-1, 0)
+        actual = self.interval - 1
+        self.assertEqual(expected, actual)
+
+        expected = Interval(-1, 0)
+        actual = 1 - self.interval
```

Comment:
I don't think this is right -- `[0, 1) - 1` should be `[-1, 0)`, but `1 - [0, 1)` should be `[1, 0)`, which is not a valid interval.